preview/MsixCore/msixmgr/msixmgr.rc

```diff
@@ -57,6 +57,34 @@ IDI_ICONBIG             ICON                    "icon1.ico"
 IDI_ICONSMALL           ICON                    "icon2.ico"
 
 
+/////////////////////////////////////////////////////////////////////////////
+//
+// Version

```

Comment:
you may need to manually edit this not to change

Reply:
Updated.